ssl/t1_lib.c

```diff
@@ -275,8 +275,9 @@ static const unsigned char ecformats_default[] = {
 
 /* The default curves */
 static const unsigned char eccurves_default[] = {
+    /* Prefer X25519 which has the fastest and most secure implementations. */
```

Comment:
It's not the fastest for us because we don't have any asm yet. I'd say "most robust". On an x86_64:

$ openssl speed ecdhx25519

                              op      op/s
 253 bit ecdh (X25519)   0.0001s   7138.5


$ openssl speed ecdhp256
                              op      op/s
 256 bit ecdh (nistp256)   0.0001s  12301.9